dApps.md

```diff
@@ -0,0 +1,46 @@
+<p align="center">
+  <img 
+    src="http://res.cloudinary.com/vidsy/image/upload/v1503160820/CoZ_Icon_DARKBLUE_200x178px_oq0gxm.png" 
+    width="125px"
+  >
+</p>
+
+<h1 align="center">NEO dApps</h1>
+
+<p align="center">
+  A curated list of dApps for Neo, in multiple stages of development.<br/>
+</p>
+
+## Contents
+
+- [Ideas for projects](#ideas-for-projects)
+- [Looking for developers](#looking-for-developers)
+- [In development](#in-development)
+- [Finished dApps](#finished-dapps)
+
+---
+
+## Ideas for projects
+
+- [Example](https://linkhere) - A dApp to keep track of your diet.
+- [Example](https://linkhere) - A dApp that makes it possible to design tokens for Neo.
+
+## Looking for developers
+
+- [Example](https://) - A dApp that aims to be a decentralised exchange - *looking for **front-end developers***.
+- [Example](https://) - Twitter, but blockchain - *looking for influential people to test*.
+
+## In development
+
+- [Example](https://) - Like YouTube, but all streamers are CoZ members - [*link to trello*](https://).
+- [Example](https://) - Aims to replace Slack with blockchainy goodness - [*link to trello*](https://).
+
+## Development halted
+
+- [Example](https://) - A physical version of the blockchain - *currently redesigning life choices*.
+- [Example](https://) - Online webstore akin to Ikea - *Ikea went blockchain, sued team*.
+
+## Finished dApps
+
+- [Example](https://) - GitHub but on Neo, [**winner of the first CoZ dApps competition**](https://linktocertificateorsomethinglikethat.com).
```

Comment:
Are you going to add URLs to each of these link blocks?

Reply:
These were just placeholders to show what it would look like when dApps get listed. Since I don't know any real dApps, the list would've been empty otherwise.  
Should I remove the links for now? Or just replace them with clear placeholders?

Reply:
I think there should only be links if the dApp actually exist.

Reply:
Similar to how this site lists all dApps for ETH https://dapps.ethercasts.com/